src/main/java/Storage.java

```diff
@@ -0,0 +1,92 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles the reading and writing of information to an external file.
+ */
+public class Storage {
+    private String filepath;
+
+    private ArrayList<Task> tasks;
+    
+    private Parser parser;
+
+    Storage(String filepath) {
+        this.filepath = filepath;
+        tasks = new ArrayList<>();
+        parser = new Parser();
+    }
+
+    /**
+     * Reads the existing txt file and stores the text into Task objects.
+     * If the file does not exist, it creates a new file.
+     * @return the list of tasks that contains the task objects.
+     */
+    public ArrayList<Task> load() {
+        try {
+            File myFile = new File("duke.txt");
+            if (myFile.createNewFile()) {
+                System.out.println("File created: " + myFile.getName());
+            } else {
+                System.out.println("File already exists");
+                Scanner myReader = new Scanner(myFile);
+                while (myReader.hasNextLine()) {
+                    String type = myReader.nextLine();
+                    if (type.contains("[T]")) {
```

Comment:
Would be good if the Tasks themselves could parse the string to create back the instance, e.g.

```java
if (type.contains("[T]")) {
    Task todo = ToDo.createFromStoredData(type);
} else if (type.contains("[D]") {
    Task todo = Deadline.createFromStoredData(type);
} else if ... {
    ...
}
```

Reply:
So the code would be more readable :)

Reply:
Noted. Thank you for the feedback.